terraform/aws-linux/main.tf

```diff
@@ -0,0 +1,137 @@
+variable "num_nodes" {
+	default = 1
+}
+
+variable "hostname_formatter" {
+	default = "amz-%d"
+}
+
+variable "ssh_user" {
+	default = "gila"
+}
+
+variable "ssh_key" {
+	default = "/home/gila/.ssh/id_rsa.pub"
+
+}
+
+variable "private_key_path" {
+	default = "/home/gila/.ssh/id_rsa"
+
+}
+
+variable "disk_size" {
+	default     = 53687091200
+}
+
+# I'd recommond to copy this locally and change the path
+variable image_source {
+  default = "https://cdn.amazonlinux.com/os-images/2.0.20200602.0/kvm/amzn2-kvm-2.0.20200602.0-x86_64.xfs.gpt.qcow2"
+}
+
+provider "libvirt" {
+  uri = "qemu:///system"
+}
+
+# a pool is where we store all images and cloud-init iso's
+resource "libvirt_pool" "aws-pool" {
+  name = "aws-pool"
+  type = "dir"
+  path = "/images"
+}
+
+# wget https://cdn.amazonlinux.com/os-images/2.0.20200602.0/kvm/amzn2-kvm-2.0.20200602.0-x86_64.xfs.gpt.qcow2
```

Comment:
Perhaps move the comment to the image_source section and point to https://cdn.amazonlinux.com/os-images/latest/kvm/ as the URL to get the latest image?